modules/assisted-installer-configuring-a-static-network.adoc

```diff
@@ -0,0 +1,30 @@
+// This is included in the following assemblies:
+//
+// assisted-installer-installing.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-a-static-network_{context}"]
+= Optional: Configuring a static network
+
+The {ai-full} supports IPv4/IPv6 dual stack networking and NMState-based static IP addressing.
```

Comment:
```suggestion
The {ai-full} supports IPv4/dual stack networking and NMState-based static IP addressing.
```

Reply:
since we don't support pure ipv6 networking

Reply:
Could we rewrite as e.g. `IPv4 and dual-stack` networking instead? I find `IPv4/dual stack` a bit confusing, i.e. the part that uses slash instead of "and"

Reply:
From previous feedback, I already fixed that and renamed the module to "Configuring host network interfaces" since we can also configure vLANs and bonding among other things.